core/src/lib.rs

```diff
@@ -57,6 +57,14 @@ pub trait DocumentBuilder {
     ) -> Result<Presentation, Box<dyn std::error::Error>> {
         Ok(Presentation::new(CONTEXT_CREDENTIALS, credentials))
     }
+
+    fn restore_identity(
+        alias: String,
```

Comment:
The alias should not be necessary in this context? The alias in the wallet cli was necessary because of storage reasons

Reply:
And the idea here is that neither in Creation Identity nor in Restore Identity we won't use storage, unlike walled-cmd in gomono, right? It is the responsibility of the caller, so I'm not counting on storage here

Reply:
Yeah exactly